demisto_sdk/tests/test_files/format_layout-copy.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
renaming this file to `format_layout_dev.json` can make it easier to understand what its contents are when looking at files under this folder